spec/requests/api_v1_work_spec.rb

```diff
@@ -112,6 +112,14 @@
                                          "work_type" => "GenericWork",
                                          "workflow_status" => nil)
       end
+
+      it 'returns work json with data when it exists' do
+        abstract = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi at tincidunt nisl. Nulla et lacus consequat, interdum eros nec, pulvinar."
+        work.abstract = abstract
+        work.save
+        get "/api/v1/tenant/#{account.tenant}/work/#{work.id}"
+        expect(json_response).to include("abstract" => [abstract])
+      end
```

Comment:
I think this might be cleaner in a context:
```suggestion
      context 'with data when it exists' do
        let(:work) { create(:work, visibility: 'open', abstract: abstract) }
        let(:abstract) { 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi at tincidunt nisl. Nulla et lacus consequat, interdum eros nec, pulvinar.' }

        it 'returns work json' do
          get "/api/v1/tenant/#{account.tenant}/work/#{work.id}"
          expect(json_response).to include("abstract" => [abstract])
        end
      end
```